cruise-control/src/yaml/responses/progressResult.yaml

```diff
@@ -15,14 +15,14 @@ StepProgress:
   required:
     - step
     - description
-    - timeInMs
+    - time-in-ms
```

Comment:
I see that the current parameter name is `time-in-ms` -- were we missing the sanity check in the earlier version about name verification?

Reply:
same as above, the [latest commit](https://github.com/linkedin/cruise-control/pull/1025/commits/fbfa8eebc29003e064dd24da6f4d32198f84903f) make the change in both places at the same time.

Reply:
Was this due to a backwards incompatible change in a previous commit? Should we change it back to its original version (if this was unintentional)?

Reply:
In #1031, we checked in some `yaml` files which has the wrong key(e.g. `timeInMs`) defined, it is missed in review. 

In this patch, since we add the sanity check for response,  `gradle build` failed and I noticed these inconsistency. 

My initial fix in this PR is to change the field in `Java` class(i.e. real response) since I feel the key in `yaml` keys look better(standard camel format). But it is back-incompatible, so in latest commit of this PR, I change the field in `yaml` instead.

 So there is no backwards incompatible introduced by previous commits, it is bug in `yaml` files.

Reply:
Got it -- thanks!